### DISCUSSION TITLE
Upgrade OpenSSH windows from  V9.5.0.0 to V9.8.1.0

### DISCUSSION THREAD

**Discussion by softwallch** [PowerShell/Win32-OpenSSH#2311] [Q&A]
Hello
how do I upgrade from OpenSSH version V9.5.0.0 to V9.8.1.0
I use the .zip installations files.
Can I simply replace the files?

Regards

**Comment by tgauth** [comment]
Yes, you could just replace the files. 

You could also use the `uninstall-sshd.ps1` script for version v9.5.0.0 then run the `install-sshd.ps1` script for version v9.8.1.0.
This would allow you to keep the binaries for both versions on the machine.
